pkg/systemd/overview-cards/realmd.jsx

```diff
@@ -187,6 +187,12 @@ export class RealmdClient {
         if (cockpit.transport.host !== "localhost")
             return Promise.resolve();
 
+        const server_sw = find_detail(realm, "server-software");
+        if (server_sw !== "ipa") {
+            console.log("cleaning up ws credentials not supported for server software", server_sw);
+            return Promise.resolve();
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19040-20230628-152825-8c7822ee-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)